src/main/java/Duke.java

```diff
@@ -1,10 +1,34 @@
+/**
+ * Main class for Duke program.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    /**
+     * Entry point of the duke program. Read from storage and creates a UI class instance.
+     * Serves as intermediate between the UI and parser.
```

Comment:
This is incorrect as there should be an empty line between the description and parameter. I have noticed similar issues in several other places too.